examples/basics/demo_tsne.py

```diff
@@ -0,0 +1,90 @@
+r"""
+TSNE embedding of the swiss roll dataset
+========================================
+
+We show how to compute a TSNE embedding with TorchDR on the swiss roll dataset.
+
+"""
+
+# Author: Rémi Flamary <remi.flamary@polytechnique.edu>
+#
+# License: BSD 3-Clause License
+
+
+# %%
+import matplotlib.pyplot as plt
+from sklearn.datasets import make_swiss_roll
+
+from torchdr import TSNE
+
+# %%
+# Load toy images
+# ---------------
+#
+# First, let's load 5 classes of the digits dataset from sklearn.
```

Comment:
change the description of the dataset to swiss roll

Reply:
good catch